metpy/calc/thermo.py

```diff
@@ -1038,3 +1038,63 @@ def most_unstable_parcel(p, temperature, dewpoint, heights=None,
     theta_e = equivalent_potential_temperature(p_layer, T_layer)
     max_idx = np.argmax(theta_e)
     return p_layer[max_idx], T_layer[max_idx], Td_layer[max_idx]
+
+
+@exporter.export
+@check_units('[pressure]', '[temperature]', '[temperature]', '[temperature]')
+def sbcape_sbcin(pressure, temperature, dewpt, profile=None):
```

Comment:
I don't love the spelling here. I'm thinking maybe `sb_cape_cin` or `sbcape_cin` to be more consistent with `cape_cin`. 

Also, since this is a surface based wrapper, there is no reason to have a profile kwarg that I can see. I'm open to be convinced, but otherwise it really isn't a wrapper that adds information.